transport.go

```diff
@@ -92,6 +94,41 @@ func getRequestBodyFromEvent(event *Event) []byte {
 	return nil
 }
 
+func getEnvelopeFromBody(body []byte) *bytes.Buffer {
+	var b bytes.Buffer
+	fmt.Fprintf(&b, `{"sent_at":"%s"}`, time.Now().UTC().Format(time.RFC3339Nano))
```

Comment:
```suggestion
func getEnvelopeFromBody(body []byte, now time.Time) *bytes.Buffer {
	var b bytes.Buffer
	fmt.Fprintf(&b, `{"sent_at":"%s"}`, now.UTC().Format(time.RFC3339Nano))
```

🌟 tah-dah, now we have a pure function that is easy to test. Given an input, the desired output is deterministic.

Reply:
Wow yes, I should have seen that way earlier 🤦. Fixed.

Reply:
No worries, team work 🦾